docs/apm/service-overview.asciidoc

```diff
@@ -2,4 +2,108 @@
 [[service-overview]]
 === Service overview
 
-Selecting a <<services,*service*>> brings you to the *Service overview*.
\ No newline at end of file
+The *Service overview* page contains a wide variety of charts and tables that provide
+visibility into how a service performs across your infrastructure. To help you isolate
+potential issues, select a <<services,*service*>> to bring you to the *Service overview* page.
```

Comment:
The working feels a bit strange as-is. What about this?

```suggestion
Selecting a <<services,*service*>> brings you to the *Service overview*.
The *Service overview* contains a wide variety of charts and tables that provide
visibility into how a service performs across your infrastructure.
```

Reply:
Looks good. Thanks.